docs/installation.md

```diff
@@ -72,97 +133,37 @@ This is to ensure your logs are tagged with relevant metadata.
-  This is the name you will see for the cluster in Sumo Logic. Default is `kubernetes`.
-  Whitespaces in the cluster name will be replaced with dashes.
-
-To install the chart, first add the `sumologic` private repo:
+Before installing the chart, you need to add `sumologic` Helm repo:
 
 ```bash
 helm repo add sumologic https://sumologic.github.io/sumologic-kubernetes-collection
+helm repo update
 ```
 
-Next you can prepare `user-values.yaml` with configuration.
-An example file with the minimum configuration is provided below.
-
-```yaml
-sumologic:
-  accessId: ${SUMO_ACCESS_ID}
-  accessKey: ${SUMO_ACCESS_KEY}
-  clusterName: ${MY_CLUSTER_NAME}
-```
-
-Now you can run `helm upgrade --install` to install our chart.
-The following command will install the Sumo Logic chart with the release name `my-release`
-in the namespace your `kubectl` context is currently set to.
-
-```bash
-helm upgrade --install my-release sumologic/sumologic \
-  -f user-values.yaml
-```
-
-> __Note__: If the release exists, it will be upgraded, otherwise it will be installed.
+## Prepare minimal configuration
 
-If you wish to install the chart in a different existing namespace you can do the following:
+Next you can prepare `user-values.yaml` with configuration.
 
-```bash
-helm upgrade --install my-release sumologic/sumologic \
-  --namespace=my-namespace \
-  -f user-values.yaml
-```
+### Required parameters
 
-If the namespace does not exist, you can add the `--create-namespace` flag.
+The Helm chart installation requires two parameter overrides:
 
-```bash
-helm upgrade --install my-release sumologic/sumologic \
-  --namespace=my-namespace \
-  --create-namespace \
-  -f user-values.yaml
-```
+- `sumologic.accessId` - Sumo [Access ID](https://help.sumologic.com/docs/manage/security/access-keys/).
+- `sumologic.accessKey` - Sumo [Access key](https://help.sumologic.com/docs/manage/security/access-keys/).
 
-If you want to override the names of the resources created by the chart,
-see [Overriding chart resource names with `fullnameOverride`](best-practices.md#overriding-chart-resource-names-with-fullnameoverride).
+### Proxy
 
-### Authenticating with container registry
+If you are installing the collection in a cluster that requires proxying outbound requests,
+please see the following [additional properties](installing-behind-proxy.md) you will need to set.
 
-Sumo Logic container images used for collection are currently hosted on
-[Amazon Public ECR][aws-public-ecr-docs] which requires authentication to provide
-a higher quota for image pulls.
-To find a comprehensive information on this please refer to
-[Amazon Elastic Container Registry pricing][aws-ecr-pricing].
+### Installing with existing Prometheus
 
-Please refer to
-[our instructions](/docs/working-with-container-registries.md#authenticating-with-container-registry)
-on how to provide credentials in order to authenticate with Docker Hub.
+If there Prometheus is already in your cluster,
```

Comment:
```suggestion
If you already have Prometheus installed in your cluster, and would like to use it instead of the Prometheus instance provided by the Chart,
```